language/bert/colotensor/dataset/wikitext.py

```diff
@@ -48,26 +48,23 @@ def seed_worker():
         torch.manual_seed(worker_seed)
         random.seed(worker_seed)
 
-    train_sampler = DistributedSampler(tokenized_dataset["train"], shuffle=True) if world_size > 1 else None
-    train_data = DataLoader(
-        tokenized_dataset["train"],
-        shuffle=(train_sampler is None),
-        sampler=train_sampler,
-        drop_last=True,
-        collate_fn=default_data_collator,
-        worker_init_fn=seed_worker,
-        batch_size=batch_size,
-        pin_memory=True,
-    )
-    test_sampler = DistributedSampler(tokenized_dataset["validation"], shuffle=False) if world_size > 1 else None
-    test_data = DataLoader(
-        tokenized_dataset["validation"],
-        sampler=test_sampler,
-        drop_last=True,
-        collate_fn=default_data_collator,
-        worker_init_fn=seed_worker,
-        batch_size=batch_size,
-        pin_memory=True,
-    )
+    data_collator = DataCollatorForLanguageModeling(tokenizer=tokenizer, mlm=True, mlm_probability=0.15)
```

Comment:
This does not include NSP task of BERT pretraining. I have written a script to deal with the nsp dataset and do bert pretraining. The links are 
- https://github.com/hpcaitech/ColossalAI-Examples/tree/main/language/bert/preprocessing
- https://github.com/hpcaitech/ColossalAI-Examples/tree/main/language/bert/zero

Reply:
ok. I will change the model.